main.go

```diff
@@ -218,7 +218,8 @@ func main() {
 		)
 
 		if err != nil {
-			log.Fatalln("Failed to close pull request")
+			log.Printf(reason)
```

Comment:
As it's generally harder to find unformatted code, could you please format it with `Reason: %s\n`?

Reply:
Done.